### ISSUE TITLE
Which licence?

### ISSUE THREAD

**Issue by SimonLab** [FAC-GM/refonte#3]
There are a lot of choice concerning the licence.
Which one should we use? What are the differences?

[Free vs Open](http://www.gnu.org/philosophy/free-software-for-freedom.en.html)

**Comment by iteles** [comment]
As part of this organisation, it will have to fall under the same license (as it ultimately won't belong to the dev team). Worth moving out now if you're considering a different license :+1:

**Comment by SimonLab** [comment]
I totally agree that the code doesn't belong to the developers and that anybody can access it and reuse it.  

Each time I create a new package with npm there is the [ISC](https://opensource.org/licenses/ISC) licence attached by default and I guess I was wondering which one of the free, open licence we should  use. Also this issue is a reminder for me to read a bit more on all the difference between these licences. I check quickly on the dwyl Agreement but I couldnt' find the name of a licence, or is the agreement is a licence on itself?

**Comment by SimonLab** [comment]
see https://github.com/dwyl/intellectual-property
